### PR TITLE
Many docstring cleanups.

### PR DESCRIPTION
**Pull request by anntzer**
Mostly D401 ("imperative mood"), and a few others.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/tri/triinterpolate.py [hunk 1]
```diff
@@ -287,7 +286,7 @@ def _interpolate_single_key(self, return_key, tri_index, x, y):
 
 class CubicTriInterpolator(TriInterpolator):
     r"""
-    A CubicTriInterpolator performs cubic interpolation on triangular grids.
+    Cubic interpolator on triangular grids.
```
Comment (timhoffm) [inline lib/matplotlib/tri/triinterpolate.py:289]:
```suggestion
    Cubic interpolator on a triangular grid.
```
Like in `LinearTriInterpolator` above.

**Comment by anntzer** [comment]
handled, and put in a few more.

File: lib/matplotlib/__init__.py [hunk 2]
```diff
@@ -192,7 +192,7 @@ def _ensure_handler():
 
 def set_loglevel(level):
     """
-    Sets the Matplotlib's root logger and root logger handler level, creating
+    Set the Matplotlib's root logger and root logger handler level, creating
```
Comment (jklymak) [inline lib/matplotlib/__init__.py:195]:
```suggestion
    Set Matplotlib's root logger and root logger handler level, creating
```

**Comment by anntzer** [comment]
handled

**Pull request merged**